galter_subjects_utils/contrib/mesh/scheme.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 Northwestern University.
+#
+# galter-subjects-utils is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""MeSH scheme."""
+
+from galter_subjects_utils.scheme import Scheme
+
+
+class MeSHScheme(Scheme):
+    """LCSH subjet kind."""
```

Comment:
Small typo and should this be MeSH instead of LCSH?

Reply:
Yes it should be MeSH. Thanks!